manifests/g/Gitlab/gitlab-runner/13.2.0/Gitlab.gitlab-runner.yaml

```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: Gitlab.gitlab-runner
+PackageVersion: 13.2.0
+PackageLocale: en-US
+Publisher: Gitlab
+PackageName: gitlab-runner
+License: MIT
+LicenseUrl: https://gitlab.com/gitlab-org/gitlab-runner/-/blob/main/LICENSE
+PackageUrl: https://gitlab.com/gitlab-org/gitlab-runner
+Moniker: gitlab-runner
+ShortDescription: GitLab Runner is the open source project that is used to run your CI/CD jobs and send the results back to GitLab
+Installers:
+  - Architecture: x86
+    InstallerType: exe
+    InstallerUrl: https://gitlab-runner-downloads.s3.amazonaws.com/v13.2.0/binaries/gitlab-runner-windows-386.exe
+    InstallerSha256: 6E91FB9FBF0548BA45DDAEF64C1DFCD06CA98ADAAF7E261E26F1F2503CC88C6B
+    InstallerSwitches:
+        Silent: install
+        SilentWithProgress: install
+ManifestType: singleton
```

Comment:
Should be updated to the multi-manifest format

Reply:
I tired to split them. Have some trouble validating them locally (`winget validate`). Can you hint me a resource in which the multi-manifest format is described? I only found https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Cinstaller-example#multiple-manifest-files